src/main/java/io/logz/logback/LogzioLogbackAppender.java

```diff
@@ -164,10 +164,15 @@ public void setInMemoryLogsCountCapacity(long inMemoryLogsCountCapacity) {
         this.inMemoryLogsCountCapacity = inMemoryLogsCountCapacity;
     }
 
+    @Deprecated
     public long getInMemoryLogsCountLimit() {
         return inMemoryLogsCountCapacity;
     }
 
+    public long getInMemoryLogsCountCapacity() {
+        return inMemoryLogsCountCapacity;
```

Comment:
So only the method name is different but they return the same variable?

Reply:
Yes. It's an old type. One is annotated as deprecated so the user will know we will delete it in the future